OpenRA.Platforms.Default/Sdl2PlatformWindow.cs

```diff
@@ -405,6 +405,12 @@ public void SetHardwareCursor(IHardwareCursor cursor)
 				SDL.SDL_ShowCursor((int)SDL.SDL_bool.SDL_FALSE);
 		}
 
+		public void SetApplicationTitle(string title)
```

Comment:
For consistency, this method name should be updated to WindowTitle as well.

Reply:
Renamed it everywhere.